src/commands.rs

```diff
@@ -186,6 +185,10 @@ pub struct WalletOpts {
     #[cfg(feature = "electrum")]
     #[arg(env = "ELECTRUM_BATCH_SIZE", short = 'b', long, default_value = "10")]
     pub batch_size: usize,
+    ///Electrum validate domain option.
+    #[cfg(feature = "electrum")]
+    #[arg(env="VALIDATE_DOMAIN",long = "validate-domain", action = clap::ArgAction::Set, default_value_t = true)]
+    pub validate_domain: bool,
```

Comment:
a user do not need to specifically indicate whether the domain should be validated